zenodo/modules/pages/config.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2015, 2016 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Configuration for Zenodo Pages."""
+
+from __future__ import absolute_import, print_function
+
+#: Description minimum length
+DESCRIPTION_MIN_LENGTH = 50
+
+#: Description maximum length
+DESCRIPTION_MAX_LENGTH = 250
+
+#: Email body template.
+EMAIL_BODY_TEMPLATE = 'zenodo_pages/email_body.html'
+
+#: Email title template.
+EMAIL_TITLE_TEMPLATE = 'zenodo_pages/email_title.html'
+
+#: Email address for support.
+SUPPORT_EMAIL = 'info@zenodo.org'
```

Comment:
Please have a look https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/config.py#L89. I think, I am not getting TypeError.